pkg/apply/task/apply_task.go

```diff
@@ -187,9 +197,9 @@ func newApplyOptions(eventChannel chan event.Event, serverSideOptions common.Ser
 	}, nil
 }
 
-func (a *ApplyTask) sendTaskResult(taskContext *taskrunner.TaskContext, err error) {
+func (a *ApplyTask) sendTaskResult(taskContext *taskrunner.TaskContext) {
 	taskContext.TaskChannel() <- taskrunner.TaskResult{
-		Err: err,
+		Err: nil,
```

Comment:
We should just avoid setting the `Err` property here instead of setting it to `nil`.

Reply:
fixed